lib/ExpensiMark.js

```diff
@@ -291,14 +292,16 @@ export default class ExpensiMark {
      * Replaces markdown with html elements
      *
      * @param {String} text
+     * @param {String[]} rules
      *
      * @returns {String}
      */
-    replace(text) {
+    replace(text, rules = []) {
```

Comment:
```suggestion
    replace(text, filterRules = []) {
```
What rules? define the param in a comment.